openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/7-csi-functional/LCWN-csi-volume-scaledown/csi-volume-scaledown

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+
+
+pod(){
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-konvoy && bash openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/7-csi-functional/LCWN-csi-volume-scaledown/csi-volume-scaledown node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"' '"'$RELEASE_TAG'"'
+}
+
+node() {
+source ~/.profile
+gittoken=$(echo "$github_token")
+job_id=$(echo $1)
+pipeline_id=$(echo $2)
+commit_id=$(echo $3)
+releaseTag=$(echo $4)
+case_id=LCWN
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+bash openebs-konvoy-e2e/utils/pooling jobname:csi-volume-scaleup
```

Comment:
@w3aman  why this scaledown is waiting for scaleup ,these two applications are different it can be run as a parallel

Reply:
My bad, actually I created PR keeping in mind that first we will scale-up and then scale down on same volume. But here we are using e2e flow...need to.change a bit
Thanks @nsathyaseelan